ethereum/core/src/test/java/org/hyperledger/besu/ethereum/vm/operations/TStoreEVMOperationTest.java

```diff
@@ -0,0 +1,452 @@
+/*
+ * Copyright Besu Contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.hyperledger.besu.ethereum.vm.operations;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.hyperledger.besu.config.StubGenesisConfigOptions;
+import org.hyperledger.besu.datatypes.Address;
+import org.hyperledger.besu.ethereum.core.AddressHelpers;
+import org.hyperledger.besu.ethereum.core.ByteCodeBuilder;
+import org.hyperledger.besu.ethereum.core.ByteCodeBuilder.Operation;
+import org.hyperledger.besu.ethereum.core.TestCodeExecutor;
+import org.hyperledger.besu.ethereum.mainnet.MainnetProtocolSchedule;
+import org.hyperledger.besu.ethereum.mainnet.ProtocolSchedule;
+import org.hyperledger.besu.evm.frame.MessageFrame;
+import org.hyperledger.besu.evm.frame.MessageFrame.State;
+import org.hyperledger.besu.evm.internal.EvmConfiguration;
+
+import org.apache.tuweni.units.bigints.UInt256;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import org.junit.runners.Parameterized.Parameter;
+import org.junit.runners.Parameterized.Parameters;
+
+@RunWith(Parameterized.class)
+public class TStoreEVMOperationTest {
```

Comment:
This unit test should live in the EVM module as we are testing the operation itself.  Tests testing the configuration enabling TSTORE should be split out and can remain.

Reply:
Migrating these tests will create a circular dependency unless I move `test-support` out of `ethereum` into a separate module. I'm happy to do this, but I think that this is better suited for a separate PR as this will introduce a lot of small changes across many files. WDYT?

Reply:
I support making that a separate PR